sdk/storage/azure-storage-blobs/test/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
```

Comment:
yes, I like this test and sample refactoring for Storage. But it needs to be a separated and visible change in our records.
Let's not smuggle it with the adding packages to vcpkg